content/markdown/stratmap/elevation-lidar.md

```diff
@@ -3,34 +3,35 @@ title: Elevation – Lidar
 layout: stratmap/main.njk
 mainimage: https://tnris-org-static.s3.amazonaws.com/images/lidar_banner.jpg
 ---
+
 <div class="container">
-  <div class="row">
-    <div class="col-md-8">
-      <p class="lead"><strong>Lidar - Light Detection and Ranging,</strong> is a remote sensing technique that utilizes light in the form of a rapidly pulsed laser to measure return distances from the Earth captured by a sensor at the source of the pulse. These combined pulse return measurements with additional spatial and temporal data recorded by the acquisition system (airborne or terrestrial) produce a three-dimensional (3-D), detailed representation of the shape of the Earth illuminating its surface characteristics.</p>
-    </div>
-    <div class="col-md-4">
-      <h3>Lidar Specifications</h3>
-      <a class="btn btn-lg btn-success" href="https://tnris-org-static.s3.amazonaws.com/documents/lidar_spec.pdf"><i class="glyphicon glyphicon-document"></i> Download Lidar Specifications</a>
-      <h3>Download Lidar</h3>
-      <a class="btn btn-lg btn-tnris" href="https://data.tnris.org"><img style="width: 20px; margin-bottom: 0 !important;" src="https://tnris-org-static.s3.amazonaws.com/images/baseline_view_comfy_white_36dp.png"> Launch DataHub</a>
-    </div>
-  </div>
+<div class="row">
+<div class="col-md-8">
+<p class="lead"><strong>Lidar - Light Detection and Ranging,</strong> is a remote sensing technique that utilizes light in the form of a rapidly pulsed laser to measure return distances from the Earth captured by a sensor at the source of the pulse. These combined pulse return measurements with additional spatial and temporal data recorded by the acquisition system (airborne or terrestrial) produce a three-dimensional (3-D), detailed representation of the shape of the Earth illuminating its surface characteristics.</p>
+<p class="lead"> TNRIS acquires lidar data through partnerships with other federal and state agencies through the <a href="/stratmap/stratmap-contracts"><strong>StratMap Contract</strong></a>, which operates through the <a href="https://dir.texas.gov/">Texas Department of Information Resources (TxDIR)</a>.
+<h2>TNRIS Lidar Coverage</h2>
```

Comment:
numerous nested tags in this file should be properly indented based on parent tags